hooks/pre_gen_project.py

```diff
@@ -0,0 +1,23 @@
+"""Python module name validation."""
```

Comment:
Does this run before the `post_gen_project.py`?

Reply:
Yes.